doc/sphinx/reference/onedim/discretization.md

```diff
@@ -1,5 +1,361 @@
+$$
+
+Where the values of $z$ are:
+
+$$
+z_{j+1/2} = \frac{1}{2}(z_{j} + z_{j+1})
+$$
+
+and,
+
+$$
+z_{j-1/2} = \frac{1}{2}(z_{j} + z_{j-1})
+$$
+
+The difference between these two values is:
+
+$$
+z_{j+1/2} - z_{j-1/2} = \frac{z_{j+1} - z_{j-1}}{2}
+$$
+
+Substituting these values into the central difference formula gives:
+
+$$
+\frac{d}{dz}\left(\mu \frac{dV}{dz}\right) \approx
+  \frac{\mu_{j+1/2} \frac{V_{j+1} - V_j}{z_{j+1} - z_j} -
+  \mu_{j-1/2} \frac{V_j - V_{j-1}}{z_j - z_{j-1}}}{\frac{z_{j+1} - z_{j-1}}{2}}
+$$
+
+This formula is used to discretize the second derivative term in the radial momentum
+equation. It takes this form due to the construction of a conservative scheme, meaning
+that point `j` sees the same value of the flux ( $\mu \frac{dV}{dz}$ ) on its right
+side as the point `j+1` sees on its left side. This is a conservative scheme because
+the fluxes are balanced at each point.
+
+### Boundary Conditions
+
+At the right boundary, the default boundary condition is a zero radial velocity.
+
+At the right boundary ($j=N$):
+
+$$
+F_{V,j} = V_j
+$$
+
+Expressing the boundary residual in this form will drive the Newton root finding
+algorithm to find the value of $ V_j $ that minimizes the residual, which in this case,
+the value of $ V_j $ that minimizes the relation above is $V=0$ .
+
+The same boundary condition is used at the left boundary.
+
+At the left boundary ($j=0$):
+
+$$
+F_{V,j} = V_j
+$$
+
+## Energy Equation
+
+The steady-state energy equation is described by:
+
+$$
+\rho c_p u \frac{\partial T}{\partial z} = \frac{\partial}{\partial z}
+  \left( \lambda \frac{\partial T}{\partial z} \right) -
+  \sum_k j_k \frac{\partial h_k}{\partial z} - \sum_k h_k W_k \dot{\omega}_k
+$$
+
+### Discretization
+
+- The term $ \rho c_p u \frac{\partial T}{\partial z} $ uses upwinding.
+- The second derivative term
+  $ \frac{\partial}{\partial z} \left( \lambda \frac{\partial T}{\partial z} \right)
+  $ uses a central difference approximation.
+
+These terms are discretized in the same way as was described above for the momentum
+equation for the upwinded term ( $\rho c_p u \frac{\partial T}{\partial z}$ ) and the
+second derivative term
+($\frac{\partial}{\partial z} \left( \lambda \frac{\partial T}{\partial z} \right)$).
+An additional term that needs to be discretized in this equation is the quantity that
+involves the species diffusive mass fluxes, $ j_k $ and the gradient of enthalpy,
+( $ \sum_k j_k \frac{\partial h_k}{\partial z} $ ).
+
+
+The discretized equation in residual form (all terms moved to one side) at the interior
+points in the domain is given below.
+
+$$
+F_{T,j} = -\rho_j c_p u_j \left( \frac{T_{\ell} -
+  T_{\ell-1}}{z_{\ell} - z_{\ell-1}} \right) +
+  \frac{\lambda_{j+1/2} \frac{T_{j+1} - T_j}{z_{j+1} - z_j} -
+  \lambda_{j-1/2} \frac{T_j - T_{j-1}}{z_j - z_{j-1}}}{\frac{z_{j+1} - z_{j-1}}{2}} -
+  \sum_k j_{k, j} \left( \frac{h_{k, \ell} -
+  h_{k, \ell-1}}{z_{\ell} - z_{\ell-1}} \right)
+$$
```

Comment:
The code doesn't use a molecular weight on that term. Is this because the equations assume certain units of h_k and omega, but the code has consistent units?

Reply:
The code uses the molar enthalpies and the molar rates of production. The formula in the documentation that uses the mass-based enthalpies, and so needs to be converted to a molar basis consistent with the rates of production (which is sort of funny, as there isn't even a function to get the partial enthalpies on a mass basis).